api/docs/release.dox

```diff
@@ -279,6 +279,9 @@ Further non-compatibility-affecting changes include:
  - Added a new drmemtrace analysis tool: syscall_mix, to count frequency of system
```

Comment:
The dr_simd_t and dr_mcontext_t changes, which break compatibility, need to be listed up above in the compatibility-changing list.

Reply:
Added section on size changes and addition of new SVE registers.